config/application.rb

```diff
@@ -4,7 +4,7 @@
 require 'rails'
 # Pick the frameworks you want:
 require 'active_model/railtie'
-require 'active_job/railtie'
+# require 'active_job/railtie'
```

Comment:
If you look at that file, all of the other remove Rails components are commented out as well. I think it makes sense to keep it this way so we have a record of which ones we have removed.